defs.bzl

```diff
@@ -1,121 +1,218 @@
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
-load("@build_bazel_rules_nodejs//:defs.bzl", "npm_install")
-
-
-def declare_file(ctx, filename, file_modifications):
-  f = ctx.actions.declare_file(filename)
-
-  # Removes an import that protoc-gen-ts adds that is not needed
-  for removal in ctx.attr.remove_dependencies:
-    file_modifications.append("echo \"$(grep -v \"%s\" %s)\" > %s" % (removal, f.path, f.path))
-
-  return f
+load("@build_bazel_rules_nodejs//:defs.bzl", "yarn_install")
```

Comment:
Why the need to move to yarn; I'd prefer us to keep a single module manager (npm) to reduce overhead if possible.

Reply:
I ran into an issue with the `npm_install` rule but using `yarn_install` fixes it. I've opened [this issue](https://github.com/bazelbuild/rules_nodejs/issues/416).

The `yarn_install` rule also is [recommended](https://github.com/bazelbuild/rules_nodejs/issues/403) over `npm_install`, though I'm fine with using `npm_install` if the issue gets resolved to keep things simple.

Reply:
Thanks for following up on this; I don't feel too strongly and the fact yarn is recommended by the bazel ecosystem is enough to sway me here if needs be.

Reply:
There's more context in the issue I opened, but short answer is it's an existing issue with npm. For now it seems like using `yarn_install` is the best option.